Weather App/UI/HomeVC/HomeView.swift

```diff
@@ -62,6 +57,12 @@ class HomeView : UIView {
         fatalError("init(coder:) has not been implemented")
     }
     
+    public func adjustFont(){
+        print("hF: ", humidityLabel.font)
+        print("pF: ", pressureLabel.font)
+        print("wF: ", windLabel.font)
+    }
+    
     public func updateWeatherView(weather: Weather){
```

Comment:
Why is this method `public`?

Reply:
I moved it and updating search method, into an extension. Methods need to be able to be accessed in ViewController to be called when view needs to be updated with new values. For ex:

`        homeViewModel.onSearchWeatherFetch = { [weak self] weather in
            DispatchQueue.main.async {
                self?.homeView.updateWeatherView(weather: weather)
            }
        }
`

Reply:
Is `public` keyword needed here? Would it work without it?

Reply:
Yes, it is unnecessary and not needed, I removed it in new commit.